commands/intakesetting.py

```diff
@@ -53,3 +55,15 @@ def __init__(self, intakeSubsystem: IntakeSubsystem) -> None:
 
     def execute(self) -> None:
         self.intake.setIntaking()
+
+
+class IntakeCoralProcess(ParallelCommandGroup):
```

Comment:
perhaps move this to somewhere else than as an "intake command" but more of a "robot combined commands" file? you can resolve if you don't think so